drivers/usb/usbohci_new/usbohci.c

```diff
@@ -1670,27 +1678,18 @@ OHCI_AbortTransfer(IN PVOID ohciExtension,
     POHCI_EXTENSION OhciExtension = ohciExtension;
     POHCI_ENDPOINT OhciEndpoint = ohciEndpoint;
     POHCI_TRANSFER OhciTransfer = ohciTransfer;
-    POHCI_TRANSFER TmpTransfer;
     POHCI_HCD_ED ED;
     ULONG_PTR NextTdPA;
-    POHCI_HCD_TD NextTD;
-    POHCI_HCD_TD TD;
-    POHCI_HCD_TD PrevTD;
-    POHCI_HCD_TD LastTD;
-    POHCI_HCD_TD td = NULL;
+    POHCI_HCD_TD TD, NextTD, LastTD;
```

Comment:
Maybe Td? I find all uppercase variable names confusing.

Reply:
I'd leave it as is, at least it's not camelCase. :grin: